frontend/src/components/IstioWizards/K8sRequestRouting.tsx

```diff
@@ -22,6 +23,11 @@ type State = {
   matchValue: string;
   k8sRules: K8sRule[];
   validationMsg: string;
+  headerOp: string;
+  headerValue: string;
+  filterType: string;
+  filterValue: string;
+  filters: string[];
```

Comment:
It's totally subjective but over the years I've found it quite useful to alphabetize these long lists of variable names, both when declaring, setting, etc.  Change is definitely not required (of course whenever modifying an existing list check to see if it's ordered, and maintain the ordering in that case).